target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -14,6 +14,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch1" \
 			"0@eth1" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
 		;;
+	mikrotik,routerboard-750-r2)
+		ucidef_set_interfaces_lan_wan "eth0.1" "eth1"
```

Comment:
ucidef_set_interface_wan "eth1"
should be enough, lan is set from the switch command.

Reply:
True, it's redundant. Thanks.